tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -12,6 +12,19 @@ use tracing_core::{
 
 /// A [`Layer`] that logs formatted representations of `tracing` events.
 ///
+/// ## Examples
+///
+/// ```rust
+/// # use tracing_subscriber::Layer as _;
+/// use tracing_subscriber::{fmt, registry::Registry};
+/// use tracing_subscriber::prelude::*;
+///
+/// let subscriber = fmt::Layer::default()
+///     .with_subscriber(Registry::default());
+///
+/// # tracing::subscriber::set_global_default(subscriber).unwrap();
```

Comment:
if this is going to be here, i'd prefer it to be part of the example...otherwise, it can just be removed.

Reply:
it wont compile without this, but the # makes it hidden.

Reply:
ah, it's needed for type inference or something? that makes sense.

Reply:
Yup